.travis.yml

```diff
@@ -1,13 +1,16 @@
 language: python
 sudo: false
 python:
+  - "2.6"
   - "2.7"
   - "3.3"
   - "3.4"
   - "3.5"
 install:
   - "pip install pylint"
   - "pip install -r requirements.txt"
+  - "if [ $TRAVIS_PYTHON_VERSION == '2.6' ]; then pip install unittest2; fi"
```

Comment:
Is it only test discovery that doesn't work in 2.6?

If it's only that maybe we can find a way to do discovery without pulling in unittest2.